src/workerd/api/pyodide/eval.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <workerd/jsg/jsg.h>
+
+namespace workerd::api::pyodide {
+
+class DynEvalImpl final: public jsg::Object {
+public:
+  void enableEval(jsg::Lock& js) {
+    js.setAllowEval(true);
+  }
+
+  void disableEval(jsg::Lock& js) {
+    js.setAllowEval(false);
+  }
+
+  JSG_RESOURCE_TYPE(DynEvalImpl) {
+    JSG_METHOD(enableEval);
+    JSG_METHOD(disableEval);
+  }
+};
+
+#define EW_EVAL_ISOLATE_TYPES                    \
```

Comment:
I think we can replace this entirely with `UnsafeEval` ... but if we don't, this should be `EW_PYODIDE_EVAL_ISOLATE_TYPES`